manifest.json

```diff
@@ -16,8 +16,8 @@
 	}],
 
 	"browser_action": {
-		"default_icon": "icon.png"
+		"default_icon": "new.png"
```

Comment:
Are you sure "new.png" is the best name? (just testing the comment looks good to me)